config/locales/en.yml

```diff
@@ -391,570 +408,624 @@ en:
+          href: https://www.citizensadvice.org.uk/about-us/northern-ireland/
+          show_to_nations:
+          - Northern Ireland
+        - id: '0076'
+          text: Get advice on benefits (Advice NI)
+          href: https://www.adviceni.net/
+          show_to_nations:
+          - Northern Ireland
+        - id: '0077'
+          text: Check if you’re eligible for the Discretionary Assistance Fund (Welsh
+            Government)
+          href: https://gov.wales/discretionary-assistance-fund-daf
+          show_to_nations:
+          - Wales
+        - id: '0078'
+          text: Find help with benefits (mygov.scot)
+          href: https://www.mygov.scot/benefits-support/
+          show_to_nations:
+          - Scotland
+        - id: '0079'
+          text: Find financial support for your business (Scottish Government)
+          href: https://findbusinesssupport.gov.scot/coronavirus-advice
+          show_to_nations:
+          - Scotland
     going_in_to_work:
       living_with_vulnerable:
-        title: "If you’re worried about going in to work"
+        title: If you’re worried about going in to work
         show_options:
-          - "Yes"
-          - "Not sure"
+        - 'Yes'
+        - Not sure
         items:
-          - id: "0080"
-            text: "Find out if you should be going to work"
-            href: "https://www.gov.uk/government/publications/staying-alert-and-safe-social-distancing/staying-alert-and-safe-social-distancing"
-          - id: "0081"
-            text: "What to do if you’re worried about going to work because of coronavirus (Citizens Advice)"
-            href: "https://www.citizensadvice.org.uk/work/coronavirus-if-youre-worried-about-working/"
-            show_to_nations:
-              - England
-          - id: "0082"
-            text: "What to do if you’re worried about going to work because of coronavirus (Citizens Advice Scotland)"
-            href: "https://www.citizensadvice.org.uk/scotland/work/coronavirus-if-youre-worried-about-working/"
-            show_to_nations:
-              - Scotland
-          - id: "0083"
-            text: "What to do if you’re worried about going to work because of coronavirus (Citizens Advice)"
-            href: "https://www.citizensadvice.org.uk/wales/work/coronavirus-if-youre-worried-about-working/"
-            show_to_nations:
-              - Wales
-          - id: "0084"
-            text: Get help from Citizens Advice
-            href: "https://www.citizensadvice.org.uk/about-us/northern-ireland/"
-            show_to_nations:
-              - Northern Ireland
+        - id: '0080'
+          text: Find out if you should be going to work
+          href: https://www.gov.uk/government/publications/staying-alert-and-safe-social-distancing/staying-alert-and-safe-social-distancing
+        - id: '0081'
+          text: What to do if you’re worried about going to work because of coronavirus
+            (Citizens Advice)
```

Comment:
```suggestion
          text: |
            What to do if you’re worried about going to work because of coronavirus
            (Citizens Advice)
```

Reply:
See here for why i think we shouldn't change these: https://github.com/alphagov/govuk-coronavirus-find-support/pull/222/commits/b0c5b0aafdd33f6b983cf66d96c9d34c0e9bfcf6#r430281216